plugins/generator-1.18.2/forge-1.18.2/templates/rangeditem/projectile_entity.java.ftl

```diff
@@ -179,6 +183,15 @@ public class ${name}Entity extends AbstractArrow implements ItemSupplier {
 		return entityarrow;
 	}
 
+	<#if data.ammoItem.isEmpty() && data.preserveProjectiles && data.pickupProjectiles>
+	@Override
+	protected boolean tryPickup(Player entity) {
+		if (this.pickup.equals(AbstractArrow.Pickup.ALLOWED))

```

Comment:
You can use `==` since `AbstractArrow.Pickup` is a Enum, also check indentation for `return super.tryPickup(entity);`